platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -99,19 +99,35 @@ are arranged as a compelling internal product. Autonomous delivery teams can
 make use of the platform to deliver product features at a higher pace, with 
 reduced coordination."
 
-Platforms are particularly relevant for cloud-native computing, which typically 
-separates supporting capabilities from application-specific logic more than 
-previous paradigms. In such environments resources like databases and object 
-stores, message queues and brokers, observability collectors and dashboards, 
-user directories and authentication systems, task runners and reconcilers and 
-more are managed independently and integrated into applications running in 
-containers and machines. A platform for cloud-native computing provides these to 
-many teams in ways that make them easy to integrate in applications and systems.
-
-At its most basic, a platform provides consistent experiences to application 
-developers for acquiring and using individual services such as a database system 
-or a secret store. Example use cases an enterprise can meet with a basic platform 
-include the following:
+The specific set of capabilities provided by a platform depends on the
+requirements of the platform's users. And while platforms _provide_ these
+required capabilities, it's critical to note that the platform team generally
+shouldn't _implement_ them themselves. Managed providers or dedicated teams
+should maintain implementations while the platform itself should be the thinnest
+consistent layer that meets an organization's requirements. For example, the
+simplest "platform" would be a wiki page with links to provision capabilities
+from providers. Some [[3]] have described this principle as seeking the
+"thinnest viable platform".
```

Comment:
again saying here that a platform should always aspire to stay thin isn't necessarily correct here 

in the Gartner paper for example when it talks about scaling platform teams a platform should START as thin but can evolve - suggesting it stays as thin as possible doesn't send the right message because a platform can likely evolve over time based on the business needs

Reply:
Thanks for adding this Abhinav, this is the difference in definition I was hoping to explore!

I would be uncomfortable using a new term to describe a long staging idea of starting small. Product may be new to internal platforms, but it is not new and we can add should lean on the existing work where possible. 

Therefore, I'd advocate that if we keep the wording in the space if "start small and then grow big" that we focus on that being defined at ["Minimum Viable Product"](https://en.wikipedia.org/wiki/Minimum_viable_product) on first delivery. 

I would personally prefer to see us use the term "Thinnest Viable Platform" but do so with a focus on how [Gregor Hope described the functionally as floating](https://m.youtube.com/watch?v=WaL3ZbLgMuI). Always staying as thin as possible to provide the required functionally.

Reply:
Also, I see MVP and TVP as useful principles that can and perhaps should coexist here. The ideas behind MVP shouldn't be new since starting small and working in and iterative way is what we usually do in software engineering. Perhaps the new thing is applying those ideas consistently to an internal platform product. 
Instead, TVP is something new that can help drive architectural decisions for a platform and avoid the trap of reinventing the wheel or starting all from scratch instead of embracing what the community and vendors have to offer out there.

Reply:
I adjusted the language a bit here to avoid referring to TVP explicitly since the term may have/develop it's own connotations. I added some language in the Platform Teams section to further reflect the idea of minimizing the platform layer, PTAL.

I love the idea of the "floating" platform and continually delegating more to managed service providers, though we don't have to mention it explicitly at this point. Thinking to leave this proposal as is for now and we can consider expanding it later.